pkg/operator/ceph/object/rgw.go

```diff
@@ -151,6 +158,8 @@ func (c *clusterConfig) startRGWPods(realmName, zoneGroupName, zoneName string)
 		if err != nil {
 			// Getting EPERM typically happens when the flag may not be modified at runtime
 			// This is fine to ignore
+			// No it is not: As errors in the mapKeystoneSecretToConfig function are returned
+			// directly and the situation may occur, that only part of the configuration is set
```

Comment:
Since we are keeping this as-is (https://github.com/rook/rook/pull/13807/files#r1640014550), let's remove the commented assertion here.